SPECS/kernel/config

```diff
@@ -154,11 +159,11 @@ CONFIG_TASKS_RUDE_RCU=y
 CONFIG_TASKS_TRACE_RCU=y
 CONFIG_RCU_STALL_COMMON=y
 CONFIG_RCU_NEED_SEGCBLIST=y
+CONFIG_RCU_NOCB_CPU=y
 # end of RCU Subsystem
 
 CONFIG_BUILD_BIN2C=y
-CONFIG_IKCONFIG=y
-CONFIG_IKCONFIG_PROC=y
+# CONFIG_IKCONFIG is not set
```

Comment:
keep on to allow reading config

https://cateee.net/lkddb/web-lkddb/IKCONFIG.html